ecommerce/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for ecommerce project.
+
+Generated by 'django-admin startproject' using Django 3.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+import os
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', '+4q5@%v)r94web7#1o@zvh(dob2nw8$9u+_%8lwi*#x2302$x1')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.environ.get('DJANGO_DEBUG', '') != 'False'
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'shop',
```

Comment:
```suggestion
    'shop',
    'accounts',
    'cart',
    'orders',
    'wishlist',
    'payment',
```
sao tạo nhiều app quá vậy nè
Trường hợp hệ thống lớn, phân tách ra các hệ thống nhỏ, vd: quản lý nhân viên, quản lý mua hàng, ... thì mới dùng nhiều app nha
Project của nhóm là ecomerce thì dùng 1 app là đc nha

Reply:
dạ vâng em sửa rồi ạ

Reply:
dùng config `'shop.apps.ShopConfig'` cho tường mình nhá